meta-ivi/conf/distro/poky-ivi-systemd.conf

```diff
@@ -21,6 +21,15 @@ DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 
 PREFERRED_VERSION_wayland-ivi-extension = "2.1"
```

Comment:
Not part of my PR... but just thought I'd ask @frznlogic since I saw it.  I think version 2.1 is the [only one provided](https://github.com/GENIVI/meta-ivi/tree/master/meta-ivi/recipes-graphics/wayland/) by .bb file, so is this line not needed, maybe?

Reply:
Done, it was a mistake on my part, I was experimenting with providing  multiple versions of wayland-ivi-extensions when we looked at wayland/weston/wayland-ivi-extension integration versions.